src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/core/AbstractCraftingRequestResolver.java

```diff
@@ -107,33 +109,46 @@ public boolean canResolveForBuilding(@NotNull final IRequestManager manager, @No
 
     /**
      * Method to check if a crafting cycle can be created.
+     *
      * @param manager the manager.
      * @param request the request.
      * @param target the target.
      * @return true if so.
      */
-    protected boolean createsCraftingCycle(@NotNull final IRequestManager manager, @NotNull final IRequest<?> request, @NotNull final IRequest<? extends IDeliverable> target)
+    protected boolean createsCraftingCycle(
+      @NotNull final IRequestManager manager,
+      @NotNull final IRequest<?> request,
+      @NotNull final IRequest<? extends IDeliverable> target)
     {
-        return createsCraftingCycle(manager, request, target, 0);
+        return createsCraftingCycle(manager, request, target, 0, new ArrayList());
     }
 
     /**
      * Method to check if a crafting cycle can be created.
+     *
      * @param manager the manager.
      * @param request the request.
      * @param target the target to create.
      * @param count the itemCount.
+     * @param reqs the list of reqs.
      * @return true if possible.
      */
     protected boolean createsCraftingCycle(
-            @NotNull final IRequestManager manager,
-            @NotNull final IRequest<?> request,
-            @NotNull final IRequest<? extends IDeliverable> target,
-            final int count)
+      @NotNull final IRequestManager manager,
+      @NotNull final IRequest<?> request,
+      @NotNull final IRequest<? extends IDeliverable> target,
+      final int count,
+      final List reqs)
```

Comment:
None generic list!

Reply:
We got a generic request, I can only put them in generic lists =D

Reply:
What are you talking about?
You added the parameter, you know the types you put in it, specify the damn types will yeah

Reply:
Weird, Intellij wasn't giving me the type of this, I had to manually search it out and put it in...

Reply:
Intellij told me it was generic "?"

Reply:
I pushed the fix anyway